test/vm_tests/builder/codegen_test.dart

```diff
@@ -658,8 +658,8 @@ main() {
 
       group('and generates props config for function components', () {
```

Comment:
Can we add some test cases for `uiForwardRef` (and other arbitrary functions, assuming we support them as per my comments above), since only `uiFunction` is currently being tested?